Dockerfile

```diff
@@ -7,12 +7,11 @@
 FROM cgr.dev/chainguard/static:latest
 
 ARG NONROOT_UID=65532
-ARG TARGET_ARCH=amd64
 
 # 65532 is the UID of the `nonroot` user in chainguard/static.
 # https://edu.chainguard.dev/chainguard/chainguard-images/reference/static/overview/#users
 USER ${NONROOT_UID}:${NONROOT_UID}
 
-COPY --chown=${NONROOT_UID}:${NONROOT_UID} "build/bomctl-linux-$TARGET_ARCH" /bomctl
+COPY --chown=${NONROOT_UID}:${NONROOT_UID} "bomctl" /bomctl
```

Comment:
Should this be something like

```Dockerfile
COPY --chown=${NONROOT_UID}:${NONROOT_UID} "dist/bomctl_linux_${TARGET_ARCH}/bomctl" /bomctl
```

The build directory structure looks something like this (unless it changed):

```
dist
├── bomctl_darwin_amd64_v1
│   └── bomctl
├── bomctl_darwin_arm64
│   └── bomctl
├── bomctl_linux_amd64_v1
│   └── bomctl
├── bomctl_linux_arm64
│   └── bomctl
├── bomctl_windows_amd64_v1
│   └── bomctl.exe
└── bomctl_windows_arm64
    └── bomctl.exe
```